apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -972,6 +972,80 @@ const Ticket = new GQLListSchema('Ticket', {
         delete: false,
         auth: true,
     },
+    kmigratorOptions: {
+        indexes: [
+            {
+                type: 'BTreeIndex',
+                fields: ['property'],
+                name: 'ticket_property',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['number'],
+                name: 'ticket_number',
+            },
+            {
+                type: 'BTreeIndex',
+                fields: ['unitName'],
```

Comment:
nice catch

Reply:
Is it the case, when filters are only by unitName without unitType?

Reply:
Yes, this is a common case. users rarely filter by unit type (at least in CRM)